LFS171x/sawtooth-material/sawtooth-tuna/client/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Sawtooth-tuna-chain-client",
+  "version": "0.0.0",
+  "description": "A simple browser based blockchain client",
+  "main": "",
+  "scripts": {
+    "build": "./node_modules/webpack/bin/webpack.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/hyperledger/LFS171x/sawtooth-material/sawtooth-tuna.git"
```

Comment:
This is missing the education folder. Also, probably needs to be: https://github.com/hyperledger/education.git